docs/images/drowio/AKS_ScalarDB_Server_App_In_Cluster.drawio

```diff
@@ -0,0 +1,329 @@
+<mxfile host="app.diagrams.net" modified="2023-06-23T05:03:08.642Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" etag="T3Ayco6tUFRY6ghw4wqN" version="21.5.0" type="google">
```

Comment:
Can you please rename the folder from `drowio` to `drawio`?🙏

Reply:
Thank you for pointing this out!
I fixed the directory name in https://github.com/scalar-labs/scalar-kubernetes/pull/201/commits/19241d827d485c10ebaadbe32b7c5decc8e12d79.